spec/cb/clients/job_spec.rb

```diff
@@ -12,108 +12,6 @@
 
 module Cb
   describe Cb::Clients::Job do
-    describe '#search' do
```

Comment:
I'm not sure you should be deleting all of this. There is still a `search` function on Cb::Clients::Job... https://github.com/careerbuilder/ruby-cb-api/blob/remove_v1_job_search/lib/cb/clients/job.rb#L17